datakyt/db/dump_db.py

```diff
@@ -0,0 +1,36 @@
+import subprocess
+import logging
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG,
+                    format='%(asctime)s.%(msecs)03d[%(process)d:%(thread)d:%(name)s:%(lineno)d] %(levelname)s %('
+                           'message)s',
+                    datefmt='%Y-%m-%d %I:%M:%S')
+
+
+def backup_postgres_db(host, database_name, port, user, password, dest_file):
+    """
+  Backup postgres db to a file.
+  """
+    try:
+        process = subprocess.Popen(
+            ['pg_dump',
+             '--dbname=postgresql://{}:{}@{}:{}/{}'.format(user, password, host, port, database_name),
+             '-Fc',
+             '-f', dest_file,
+             '-v'],
+            stdout=subprocess.PIPE
+        )
+        output = process.communicate()[0]
+        if int(process.returncode) != 0:
+            logger.error('Command failed. Return code : {}'.format(process.returncode))
+            exit(1)
+        return output
+    except Exception as e:
+        logger.error(e)
+
+
+if __name__ == '__main__':
+
+    backup_postgres_db('localhost', 'datakyt', '5432', 'datakyt_admin', 'password', 'out2.txt')
```

Comment:
`out2.txt` seems not a proper name for the db backup

Reply:
Done

Reply:
I think that we need to specify a DB name and the date on which a dump was made in the file name. Let's make it something  like this:
<db_name><date>.sql

Reply:
Done